packages/data-context/src/actions/ProjectConfigDataActions.ts

```diff
@@ -56,34 +51,17 @@ export class ProjectConfigDataActions {
     return dfd.promise as Cypress.ConfigOptions
   }
 
-  getChildOptions (config: {
-    resolvedNodePath?: string
-    resolvedNodeVersion?: string
-   }): ForkOptions {
+  private forkConfigProcess (opts: ForkConfigProcessOptions) {
+    const configProcessArgs = ['--projectRoot', opts.projectRoot, '--file', opts.configFilePath]
+
     const childOptions: ForkOptions = {
       stdio: 'pipe',
+      cwd: path.dirname(opts.configFilePath),
       env: {
         ...process.env,
         NODE_OPTIONS: process.env.ORIGINAL_NODE_OPTIONS || '',
       },
-    }
-
-    if (config.resolvedNodePath) {
```

Comment:
We might need to open a new issues around node 17 support since removing this is a regression of 9.1.1: https://github.com/cypress-io/cypress/issues/18914